index.html

```diff
@@ -4,21 +4,15 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/7bfe064e41.js" crossorigin="anonymous"></script>
     <link rel="stylesheet" href="Style.css">
```

Comment:
- [ ] Kindly reference correctly to your stylesheet. It is <code>style.css</code> instead of the <code>Style.css</code> 🤗

Reply:
Thank you but unfortunately I have just 5 minutes left